dfastmi/__main__.py

```diff
@@ -94,30 +94,34 @@ def parse_arguments() -> Tuple[str, str, Optional[str], str, bool]:
     parser = argparse.ArgumentParser(description="D-FAST Morphological Impact.")
 
     parser.add_argument(
-        "--language", help="display language 'NL' or 'UK' (UK is default)"
+        "--language",
+        choices=["UK", "NL"],
+        default="UK",
+        help=argparse.SUPPRESS,
     )
-    parser.set_defaults(language="UK")
 
     parser.add_argument(
-        "--mode", help="run mode 'BATCH', 'CLI' or 'GUI' (GUI is default)"
+        "--mode",
+        choices=["CLI", "BATCH", "GUI"],
```

Comment:
Should CLI still be an option, since it is not noted in the help message?

Reply:
As long as it's part of the code, I would keep it as option available. However, this option is no longer promoted. The description of the option is still available in the appendix of the technical reference manual, but no longer in the user manual.

By the way, it turned out that by listing it in the choices, the "CLI" option was still visible in the help message. Furthermore, it turned out that these choices are case sensitive by default. For these reasons, I've removed the choices option in a later commit and reverted to only a help text without mentioning the CLI mode.

Reply:
Discussed with Ralph resolving.